src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-useParallel.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    <p>Select to run the tests in parallel</p>  
```

Comment:
Whether to run tests in parallel. This feature requires a Parallel Computing Toolbox license, and might not be compatible with other options. If incompatible, testing occurs in serial regardless of the value of this feature.

Reply:
Done.